app/Console/Components/Npm/FetchTotals.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Console\Components\Npm;
+
+use App\Events\Npm\TotalsFetched;
+use Developmint\NpmStats\NpmStats;
+use GuzzleHttp\Client;
+use Illuminate\Console\Command;
+
+class FetchTotals extends Command
+{
+    protected $signature = 'dashboard:fetch-npm-totals';
+
+    protected $description = 'Fetch totals for all our NPM packages';
+
+    public function handle()
+    {
+        $npmStats = new NpmStats(new Client());
+
+        $packageList = $this->getPackageList();
+
+        /*
+         * First idea was to filter the non-scoped and scoped packages so at least the
+         * scoped packages could've been processed by bulk. This would work, but the
+         * allowed range for bulk is lower than normal, which would lead to wrong,
+         * incomplete or weird data in some cases
+         *
+         */
+
+
+        $totals = $packageList->reduce(function ($carry, $package) use ($npmStats) {
```

Comment:
Try to avoid `reduce` here. Using `pipe` and `sum` will be cleaner.

Example: https://github.com/spatie/dashboard.spatie.be/blob/master/app/Console/Components/Packagist/FetchTotals.php